ConfigurationValidation.hpp

```diff
@@ -290,8 +290,8 @@
 
 #if (DEC_DRIVER_TYPE == DRIVER_TYPE_TMC2209_UART)
```

Comment:
What about the Focuser motor?

Reply:
Ah my mistake. I forgot about that one. I'll update it.